package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "@langleyfoxall/react-dynamic-data-table",
-  "version": "7.17.1",
+  "version": "7.17.2",
```

```suggestion
  "version": "7.18.0",
```

This should be a minor version change not a patch version as we're adding new functionality.